Evolutionary_loop/Loop_Parts/Part_B/Part_B_VPol_job1.sh

```diff
@@ -78,6 +78,7 @@ chmod -R 777 $XmacrosDir
 cd $XmacrosDir
 
 # frequencies are scaled up by 100 to avoid float operation errors in bash
+# note: this should match the list in part_c_vpol.py
```

Comment:
This is part of the TODO list of part_c_vpol.py;
Eventually I want to have this somehow passed as a variable; maybe when we rewrite Part_B in python this would be easier.